test/calculator.test.ts

```diff
@@ -0,0 +1,75 @@
+import { expect } from "chai";
+import { Calculator, Operation } from "../src/Calculator";
+
+describe('Calculator', () => {
```

Comment:
GG here! :clap: 

Just the parenthesis are not compliant to the [Allman](https://en.wikipedia.org/wiki/Indentation_style#Allman_style) style :ok_man:

Reply:
Should be fixed